src/py/aspen/database/models/entity.py

```diff
@@ -1,12 +1,31 @@
+from __future__ import annotations
+
 import enum
-from typing import Mapping, Union
+from typing import (
+    Mapping,
+    MutableSequence,
+    Optional,
+    Sequence,
+    Tuple,
+    Type,
+    TYPE_CHECKING,
+    TypeVar,
+    Union,
+)
 
 import enumtables
-from sqlalchemy import Column, ForeignKey
+from sqlalchemy import Column, ForeignKey, Integer
+from sqlalchemy.orm import backref, relationship
 
 from .base import base, idbase
 from .enum import Enum
 
+if TYPE_CHECKING:
+    from .workflow import Workflow
```

Comment:
why do you need this part?

Reply:
I use it in type annotations, for example [here](https://github.com/chanzuckerberg/aspen/pull/81/files#diff-c25c7ebf669612dcdc0aa054e26ff3d103f4b125ab2b05a48a63a0691357e552R67).  If I don't gate it with `if TYPE_CHECKING`, however, I get recursive imports.

Reply:
to be specific, mypy is two-pass, normal python execution is one-pass.